script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -287,27 +321,32 @@ Well, it'll depend on the proclivities of the person attending, so you'll just h
 [sha:fcbc2700ddcdb6873b68d8dc5a4217ebd5a8763d]{
 -- Page 424, Offset 36505.
 -- 趣味って……と抗議したくなるのをぐっと堪える。
-...I desperately hold back the urge to protest at the word 'proclivities'.
+'Proclivities'...?
+ I take issue with her word choice, but do my best to bite my tongue.
 }
 [sha:79f61b1044c61f1f8682a3b0f75c712164260a4d]{
 -- Page 424, Offset 36506.
 -- 先輩はたまに泣く子もドン引くクールさを垣間見せてくれる。天然か。
-Senpai sometimes seems like a crying child, but at other times, she can be startlingly cold. Is she an airhead?
+// Is this like a 泣く子も黙る modulation?
+// 天然ボケ modified a bit because 'airhead' comes off as more no thoughts head empty than speaking goofs
+Every now and again Senpai says something so ominous it would scare a crying child into behaving.
+ Does she just not realize what she's saying?
 }
 [sha:690091a8c6255ff8fa96ff8655546029715487b6]{
 -- Page 425, Offset 36507.
 -- 「問題は過程ではなく結果です。
-"The problem here isn't the process, it's the result.
+"The problem here isn't the process, it's what comes after.
 }
 [sha:45e82a5e2a1ee3ed68eb30726faf6b02024d7fb8]{
 -- Page 425, Offset 36508.
 -- 教会でも治療ができなかった場合、遠野くんは吸血種として扱われてしまうんです。
-If the Church can't do anything about it, they'll regard you as a vampire.
+If the Church isn't able to cure you, they'll write you off as a bloodsucker.
 }
 [sha:24a673e855c49731c9489efa5183d4d85678dd2c]{
 -- Page 425, Offset 36509.
 -- ……そうなってしまったら、貴方はわたしと同じ扱いを受ける事になる。死んだ方がマシな扱いを、されてしまう。
-...If that happens, you'll be subject to the same treatment that I was. Even death would be better at that point.
+If that happens... you'll be subjected to the same abuse that I was.
+ Even death would be preferable to being treated like that.
 }
 [sha:8bff0d75936c49222070ad8695c2e6b6d88a1a33]{
 -- Page 425, Offset 36510.
```

Comment:
Nope sorry that's not a natural way to extend that thought. There is nothing wrong with `that's why`!

Reply:
Conjunction 'that's why' is basically only ever an occurrence of translated "dakara", I will not budge on this point (or rather, 'that' is not a conjunction used to connect sentences). At least stick an "and" in front of it.

Reply:
I mean, sure, but it's totally valid English to do that? I search "that's why" in a newspaper corpus and it basically _always_ shows up at the start of a sentence:
https://ludwig.guru/s/that's+why

I will grudgingly put an And on it here, but I do not think there is a good reason to avoid this very normal way to say things.